registry/auth.go

```diff
@@ -180,6 +180,8 @@ func loginV1(authConfig *cliconfig.AuthConfig, registryEndpoint *Endpoint) (stri
 				}
 				// *TODO: Use registry configuration to determine what this says, if anything?
 				return "", fmt.Errorf("Login: Account is not Active. Please see the documentation of the registry %s for instructions how to activate it.", serverAddress)
+			} else if resp.StatusCode == 500 { // Issue #14326
+				return "", fmt.Errorf("Internal Server Error")
```

Comment:
@ankushagarwal It is okay to dump this to the logs. If we hadn't seen the body, we'd have no clue what the actual failure was. If you just say `index.docker.io returned a 500 while trying to authenticate`, the root cause will be impossible to track down.